oauth2client/flask_util.py

```diff
@@ -320,10 +336,11 @@ def _make_flow(self, return_url=None, **kwargs):
         kw = self.flow_kwargs.copy()
         kw.update(kwargs)
 
-        extra_scopes = util.scopes_to_string(kw.pop('scopes', ''))
-        scopes = ' '.join([util.scopes_to_string(self.scopes), extra_scopes])
+        extra_scopes = kw.pop('scopes', [])
+        scopes = util.scopes_to_string(
```

Comment:
Is this `util.scopes_to_string` call still required? The `scope` parameter of `OAuth2WebServerFlow` is documented to be "string or iterable of strings, scope(s) of the credentials being requested.", which of course is the funniest thing I've seen today since a string is already an iterable of strings. I don't think that that will ever stop being funny.